config.json

```diff
@@ -329,6 +329,11 @@
       "slug": "kindergarten-garden",
       "difficulty": 1,
       "topics": []
+    },
+    {
+      "slug": "collatz-conjecture",
+      "difficulty": 1,
```

Comment:
I love this idea and am not sure why I haven't thought of it before! I'll go ahead and make a guesstimate for this exercise :)